src/Typedefs/DependantLookup.idr

```diff
@@ -0,0 +1,25 @@
+||| This module is by itself because its only function takes ages to compile
+module DependantLookup
```

Comment:
Isn't "dependant" a noun meaning a person who depends on another? :)

Reply:
Yes, it's a frenchism, fixed